gslib/tests/testcase/integration_testcase.py

```diff
@@ -273,6 +276,26 @@ def _SetObjectCustomMetadataAttribute(self, provider, bucket_name,
       self.xml_api.PatchObjectMetadata(bucket_name, object_name, obj_metadata,
                                        provider=provider)
 
+  def get_random_ascii_chars(self, size, seed=None):
```

Comment:
[no action required]

This isn't specific to integration tests, and thus should probably go in the testing `util` module instead.  We should go through and reorganize things to be in their correct/appropriate modules at some point, but that's fairly low priority, so if it will make diffs and merge conflicts easier to deal with (in doing the rest of your cherry pick PRs), just leave it here :)

Reply:
That sounds good! I dropped it into `gsutil/utils/text_util.py` if that sounds like a reasonable new home for the function. :house_with_garden: